LayoutTests/imported/w3c/web-platform-tests/html/semantics/forms/the-input-element/pattern_attribute-expected.txt

```diff
@@ -1,5 +1,11 @@
-Pattern Attribute
+pattern attribute
 
 
-PASS pattern attribute support on input element
+PASS basic <input pattern> support
+PASS <input pattern> is Unicode code point-aware
+PASS <input pattern> supports Unicode property escape syntax
+PASS <input pattern> supports Unicode property escape syntax for properties of strings
+PASS <input pattern> supports set difference syntax
+PASS <input pattern> supports string literal syntax
+FAIL <input pattern> enables the RegExp v flag assert_true: <input pattern="[&amp;&amp;]" value="foo"> should be valid expected true got false
```

Comment:
@mathiasbynens is this an existing WebKit bug with our v implementation or is something else amiss here?

Reply:
`/[&&]/v` is meant to throw an early `SyntaxError`, since `&&` matches [the `ClassSetReservedDoublePunctuator` production](https://arai-a.github.io/ecma262-compare/?pr=2418#ClassSetReservedDoublePunctuator) in this case. This is also given as an example in the proposal README: https://github.com/tc39/proposal-regexp-v-flag#how-is-the-v-flag-different-from-the-u-flag

@msaboff, is this an implementation bug, or did you find an error in our proposed spec text?

Reply:
This is an existing bug in the WebKit implementation that I happen to be working on.

Reply:
Thanks for confirming. I’ve written some more Test262 tests explicitly capturing these cases now: https://github.com/tc39/test262/pull/3820